docs/index.md

```diff
@@ -50,6 +50,13 @@ Follow the following steps to get set up:
 
 1. Install necessary dependencies above.
 
+    ```sh
```

Comment:
Can you change the sentence above to say
"Install the necessary dependencies above. To install protobuf and boost:"